src/JSON.ts

```diff
@@ -1067,8 +1067,10 @@ function parseObject(tokenizer: Tokenizer, tokens: Token[]): ObjectValue {
             }
         } else {
             const propertyValue: Value = parseValue(tokenizer, tokens);
-            propertySpan = propertySpan.union(propertyValue.span);
-            objectSpan = objectSpan.union(propertyValue.span);
+            if (propertyValue) {
+                propertySpan = propertySpan.union(propertyValue.span);
+                objectSpan = objectSpan.union(propertyValue.span);
+            }
 
             properties.push(new Property(propertySpan, propertyName, propertyValue));
```

Comment:
Should this line also be inside `if (propertyValue) {`?

Reply:
No, we still create the property entry, it just doesn't have a property value token. Someday I'll have to strict-null this code, that will not be fun. :-(